wiki/content/graphql/lambda/overview.md

```diff
@@ -0,0 +1,195 @@
++++
+title = "Overview"
+weight = 1
+[menu.main]
+    parent = "lambda"
+    identifier = "lambda-resolvers-overview"
++++
+
+Lambda provides a way to write your custom logic in JavaScript, integrate it with your GraphQL schema, and execute it using the GraphQL API in a few easy steps:
+
+- Setup a Dgraph cluster with a working lambda server (not required for [Slash GraphQL](https://dgraph.io/slash-graphql) users)
+- Declare lambda queries, mutations, and fields in your GraphQL schema as needed
+- Define lambda resolvers for them in a JavaScript file
+
+This also simplifies the job of UI developers, allowing them to work in JavaScript, without knowing any other backend language.
+
+Dgraph doesn't execute your custom logic itself. It makes external HTTP requests to a user-defined lambda server. That means, you can deploy your lambda logic into the same Kubernetes cluster as your Dgraph instance. 
+
+{{% notice "tip" %}}
+If you want to deploy your own lambda server, you can find the implementation of Dgraph Lambda in our [open-source repository](https://github.com/dgraph-io/dgraph-lambda). Please refer to the documentation on [setting up a lambda server](/graphql/lambda/server) for more details.
+{{% /notice %}}
+
+## Declaring lambda in a GraphQL schema
+
+There are three places where you can use the `@lambda` directive and thus tell Dgraph where to apply custom JavaScript logic.
+
+1. You can add lambda fields to your types and interfaces
+
+```graphql
+type MyType {
+    ...
+    customField: String @lambda
+}
+```
+
+2. You can add lambda queries to the Query type
+
+```graphql
+type Query {
+    myCustomQuery(...): QueryResultType @lambda
+}
+```
+
+3. You can add lambda mutations to the Mutation type
+
+```graphql
+type Mutation {
+    myCustomMutation(...): MutationResult @lambda
+}
+```
+
+## Defining lambda resolvers in JavaScript
+
+A lambda resolver is a user-defined JavaScript function that performs custom actions over the GraphQL types, interfaces, queries, and mutations. There are two methods to register JavaScript resolvers:
+
+- `self.addGraphQLResolvers`
+- `self.addMultiParentGraphQLResolvers`
+
+### addGraphQLResolvers
+
+The `self.addGraphQLResolvers` method takes an object as an argument, which maps a resolver name to the resolver function that implements it. The resolver functions registered using `self.addGraphQLResolvers` receive `{ parent, args, graphql, dql }` as argument:
+
+- `parent`, the parent object for which to resolve the current lambda field registered using `addGraphQLResolver`.
+Available only for types and interfaces (`null` for queries and mutations)
+- `args`,  the set of arguments for lambda queries and mutations (`null` for types and interfaces)
+- `graphql`, a function to execute auto-generated GraphQL API calls from the lambda server
```

Comment:
could mention that the user's authheader is passed back to the graphql function, so this can be used securely.